reviews/review-03+2020-11-30.md

```diff
@@ -0,0 +1,169 @@
+﻿1. What do you think is good about this manuscript?
+
+a. This is a solid presentation of often overlooked skills for designing not
+just useful, but successful software for scientific research. It is all too easy
+to develop software that "works" but simply working is a lower bar that most
+researchers realize. Software is essential to research---especially in the
+natural sciences---but it is a part of the research workflow that can easily
+evade scrutiny (few peer reviews include code review). This book presents a
+recipe for skills (computational and collaborative) that can help research
+software developers deliver reproducible science and build confidence that their
+work can be sustained even after their initial contributions.
+
+b. Notes/callouts and glossary are very helpful for the reader. Code
+instructions are clear, thorough, and nicely commented.
+
+2. What reservations do you have about this manuscript?
+
+a. Chapters on their own a solid. I think there are times where I felt myself
+wanting a bit more "glue" or context. For example, as currently presented,
+learning objectives are separated from chapters. I want to know at the beginning
+of a chapter: 1) How is this chapter going to make me a better RSE; 2) what are
+the specific objectives that are going to get me there. This content is all
+there, but why not make it more explicit? For example, in Chapter 4's intro you
+tell me that I may need to process thousands of files (good). But the first
+sentence comes out of nowhere (reused from some other text?) talking about GUI
+IDEs. A little bit of punchy rewriting can answer what I think is the primary
+question---How is this going to make be a better RSE.
+
+b. I wrote my answer to question 4 (comment a) before I finished the
+introduction to the first chapter. I feel like I am still not 100% convinced
+about the title. It works as is without being misleading, but I think revisions
+might focus this book a bit, and hopefully the authors will be open to change if
+they agree there are different ways to focus things.
+
+3. In terms of content and structure, do you think the chapters need any
+revisions, such as additions or omissions? Please be as specific as possible.
+
+a. Looking at the overall order of chapters---would it be beneficial to reorder
+in a way that more closely resembles how an RSE might create a project. This
```

Comment:
We literally create a project in the book, so I'm not sure you can argue that the chapter order doesn't resemble how an RSE might create a project.